src/config.ts

```diff
@@ -1,8 +1,8 @@
-module.exports = {
+export const constants = {
   bearer: 'Bearer ',
   functionAppApiPath: '/api/',
   functionAppDomain: '.azurewebsites.net',
-  functionsAdminApiPath: '.azurewebsites.net/admin/functions/',
+  functionsAdminApiPath: '/admin/functions/',
```

Comment:
are function and functions referring to 2 different things?

Reply:
This is a partial path to one of the Kudu rest endpoints to retrieve functions in a function app.